packages/react-resizable-panels/src/PanelContexts.ts

```diff
@@ -16,8 +16,3 @@ export const PanelGroupContext = createContext<{
   stopDragging: () => void;
   unregisterPanel: (id: string) => void;
 } | null>(null);
-
-// Workaround for Parcel scope hoisting (which renames objects/functions).
-// Casting to :any is required to avoid corrupting the generated TypeScript types.
-// See github.com/parcel-bundler/parcel/issues/8724
-(PanelGroupContext as any).displayName = "PanelGroupContext";
```

Comment:
Actually, this one might still be needed because JS engine can't infer a name here. Context isn't a function.